Doc/library/test.rst

```diff
@@ -186,18 +186,18 @@ top-level directory where Python was built. On Windows, executing
 tests.
 
 
-:mod:`test.test_support` --- Utility functions for tests
-========================================================
+:mod:`test.support` --- Utility functions for tests
+===================================================
 
-.. module:: test.test_support
+.. module:: test.support
    :synopsis: Support for Python regression tests.
 
 .. note::
 
-   The :mod:`test.test_support` module has been renamed to :mod:`test.support`
-   in Python 3.x.
+   The :mod:`test.support` module has been renamed to :mod:`test.support`
```

Comment:
This should start with "The `test.test_support` module..." and explicitly state that the latter has been retained as an alias in 2.7.x.